Assets/Python/CityNameManager.py

```diff
@@ -4974,6 +5046,10 @@ def onRevolution(iCiv):
 	"Urajio"		:	"Vladivostok",
 	"Urajiosutokku"		:	"Vladivostok",
 	"Vladivostok"		:	"Vladivostok",
+	"Haesamwi"		:	"Vladivostok",
+	"Hwaju"			:	"Vladivostok",
+	"Anju"			:	"Vladivostok",

```

Comment:
What are all these cities? They can't all be Vladivostok.

Reply:
There are 3 tiles marked as Vladivostok in the Russian map.
Hwaju, Anju and Jeongju are names of cities that existed in that area.
After the fall of the Balhae Kingdom that area was lost to Korea.
After Vladivostok was settled, Koreans called it Haesamwi.

In-game, if Korea settles that area the city will be named as Hwaju/Anju/Jeongju.
If they take it from another civ it will be called Haesamwi.